docs/index.md

```diff
@@ -4,14 +4,23 @@ description: EthSigner is an open-source private transaction manager developed u
 
 # EthSigner Transaction Signer
 
-EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. 
+## What is Ethsigner?
 
-EthSigner provides private key signing to separate key management from transaction validation. EthSigner supports [Hashicorp Vault](Using-EthSigner/Hashicorp.md) 
-and [Azure Key Vault](Using-EthSigner/Azure.md). 
+EthSigner is an open-source, client-agnostic, Ethereum transaction signer developed under the Apache 2.0 license and written in Java. It adds a layer of security by signing transactions using a private key that can be secured in a variety of cloud providers, or encrypted on a local disk. EthSigner therefore separates private key management from transaction validation. 
 
-## What can you do with EthSigner? 
+## How does it work?
 
-EthSigner can: 
+EthSigner acts as a proxy service by forwarding requests between the client and network. When it receives a transaction it will generate a signature using the stored private key, and forward the transaction and signature to the client to include in the blockchain.
```

Comment:
Editing comments - use present tense (ie, don't use _will_) and again may not be clear what _it_ refers to without rereading previous sentence: 

```suggestion
EthSigner acts as a proxy service by forwarding requests to the client. When EthSigner receives a transaction it generates a signature using the stored private key, and forwards the signed transaction to the client.
```

Reply:
EthSigner forwards requests to the client rather than between the client and the network